Library/Homebrew/formula.rb

```diff
@@ -1874,6 +1856,26 @@ def to_hash
     hsh
   end
 
+  # Returns the bottle information for a formula
+  def bottle_hash
+    bottle_spec = stable.bottle_specification
+    hash = {
+      "rebuild"  => bottle_spec.rebuild,
+      "cellar"   => (cellar = bottle_spec.cellar).is_a?(Symbol) ? cellar.inspect : cellar,
+      "prefix"   => bottle_spec.prefix,
+      "root_url" => bottle_spec.root_url,
+      "files"    => {},
+    }
+    bottle_spec.collector.each_key do |os|
+      bottle_url = "#{bottle_spec.root_url}/#{Bottle::Filename.create(self, os, bottle_spec.rebuild).bintray}"
+      checksum = bottle_spec.collector[os][:checksum]
+      hash["files"][os] = {
+        "url"    => bottle_url,
+        "sha256" => checksum.hexdigest,
+      }
+    end
```

Comment:
You should be returning `hash` here, no?

Reply:
Yup 👍🏻